services/transactionService.ts

```diff
@@ -42,6 +43,23 @@ export async function getTransactionAmount (transaction: BCHTransaction.AsObject
   )
 }
 
+export async function getTransactionValue (transaction: TransactionWithAddressAndPrices): Promise<QuoteValues> {
+  const ret: QuoteValues = {
+    usd: new Prisma.Decimal(0),
+    cad: new Prisma.Decimal(0)
+  }
+  if (transaction.prices.length !== 2) throw new Error(`txid${transaction.id}, ts${transaction.timestamp} ${RESPONSE_MESSAGES.MISSING_PRICE_FOR_TRANSACTION_400.message}`)
```

Comment:
It is 2 because we expect the price both in USD and in CAD, and since we always expect to have both of them, checking for equality against `2` is ideal, because no more or no less would be right.

In any case, writing the hardcoded `2` is indeed a bad practice, so I created a constant `N_OF_QUOTES` for it (check the second to last commit)